src/ziggurat/metrics.clj

```diff
@@ -76,20 +76,26 @@
    (inc-or-dec-count sign {:metric-namespace metric-namespace :metric metric :n n :additional-tags additional-tags}))
   ([sign {:keys [metric-namespace metric n additional-tags]}]
    (let [metric-ns (get-metric-namespaces metric-namespace)
-         meter     ^Meter (mk-meter metric-ns metric (get-map additional-tags))]
+         meter ^Meter (mk-meter metric-ns metric (get-map additional-tags))]
      (.mark meter (sign (get-int n))))))
 
 (def increment-count (partial inc-or-dec-count +))
 
 (def decrement-count (partial inc-or-dec-count -))
 
+(defn report-histogram
+  ([metric-namespaces val]
+   (report-histogram metric-namespaces val nil))
+  ([metric-namespaces val additional-tags]
+   (let [metric-namespace (get-metric-namespaces metric-namespaces)
+         histogram ^Histogram (mk-histogram metric-namespace "all" additional-tags)]
+     (.update histogram (int val)))))
+
 (defn report-time
```

Comment:
instead of redefining this function, why not do
(def report-time report-histogram)